src/controllers/documents/documentType_controller.go

```diff
@@ -0,0 +1,44 @@
+package documents
+
+import (
+	"github.com/go-chi/chi"
+	"net/http"
+	"obas/src/config"
+	io "obas/src/io/documents"
+	"obas/src/io/log"
+)
+
+func DocumentsType(app *config.Env) http.Handler {
+	r := chi.NewRouter()
+	r.Get("/", documentsTypeHandler(app))
+	return r
+}
+
+func documentsTypeHandler(app *config.Env) http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+		alldocst, err := io.DocumentType()
+
+		if err != nil {
+			app.ServerError(w, err)
+		}
+
+		type PageData struct {
+			logs []io.LogEvent
```

Comment:
this line must be change to logs[ ]io.DocumentType because we are calling the package documentType_io